pipelines/optimus/Optimus.changelog.md

```diff
@@ -1,3 +1,9 @@
+#optimus_v2.1.0
+
+* Removed zarr file
+
+* Added emptyDrops for sn_rna mode
```

Comment:
Didn't we actually remove emptyDrops, not add it? 

I think we need to add a bit more information about the removal of zarr.

Something like "Removed zarr formatted matrix and metrics outputs and replaced with loom" and I believe this is a breaking change. If this is going out with the features for the snRNAseq support it's definitely a major version update. Are the snRNAseq changes included in this PR?